src/components/TalentPage/Pagination.tsx

```diff
@@ -0,0 +1,24 @@
+import { FC } from 'react';
+import { Paginate } from './interfaces';
+
+const Pagination: FC<Paginate> = ({ filterPerPage, total, paginate }: Paginate) => {
+	const pageNumbers = [];
+	for (let i = 1; i <= Math.ceil(total / filterPerPage); i++) {
+		pageNumbers.push(i);
+	}
```

Comment:
? same, why we have calculculation with each rerender

Reply:
fixed